pulsar-client-admin-api/src/main/java/org/apache/pulsar/client/admin/Lookup.java

```diff
@@ -57,7 +58,7 @@
      * @param topic
      * @return the broker URLs that serves the topic
      */
-    CompletableFuture<Map<String, String>> lookupPartitionedTopicAsync(String topic);
+    CompletableFuture<Map<String, List<String>>> lookupPartitionedTopicAsync(String topic);
```

Comment:
To be clear: this is the breaking API change that I am referring to

Reply:
Sorry for the misunderstanding. I have modified the API implmention. 
The API will keep compatibility, `patitioned-lookup` result is still the type of `Map<String, String>`.
For the new display, add an option `--key-by-broker-url` which default is false.
If enalbe `--key-by-broker-url`, the result will be keyed by broekrUrl, and the value is the `List<String>.toString` which contains all partitions belong to the topic on this broker.